tests/components/modern_forms/test_config_flow.py

```diff
@@ -159,45 +157,3 @@ async def test_zeroconf_confirm_connection_error(
 
     assert result.get("type") == RESULT_TYPE_ABORT
     assert result.get("reason") == "cannot_connect"
-
-
-async def test_user_device_exists_abort(
```

Comment:
We still want to test these cases where there's an existing config entry and we try to set up the same device again.

Reply:
ok back in and skip working